### DISCUSSION TITLE
IO: Silixa .h5 format

### DISCUSSION THREAD

**Discussion by ahmadtourei** [DASDAE/dascore#152] [General]
@d-chambers 
I'm sure we cover Silixa's ".tdms" format. How about Silixa's ".h5" (the HDF5 format)? 
I previously used DASCore for Terra15's HDF5 files but I could not get patches from the spool directed to a directory containing ".h5" files.

**Comment by d-chambers** [comment]
Hey @ahmadtourei,

That's right, we haven't implemented that yet. #121 is related. Do you happen to have a small (<50 mb) silixa h5 file we can use to implement and test a parser?

**Comment by ahmadtourei** [comment]
Hey @d-chambers,

The smallest file I have is ~160 MB. I can share it with you via OneDrive. I'd be happy to help implement it.

**Comment by d-chambers** [comment]
Thanks for sending it, I just got it. I feel dumb but I forgot we already have a silixa test file (~3mb) in our test directory [here](https://github.com/DASDAE/test_data/blob/master/das/iDAS005_hdf5_example.626.h5). Silixa graciously sent this to us about a year ago.  

I have a few hours today, let me take a stab at its implementation then you can review and test it on your datasets.

**Comment by ahmadtourei** [comment]
Thanks, @d-chambers. 
I tested this IO and created a new [issue](https://github.com/DASDAE/dascore/issues/158).
